entity-service-change-event-generator/build.gradle.kts

```diff
@@ -0,0 +1,21 @@
+plugins {
+  `java-library`
+  jacoco
+  id("org.hypertrace.jacoco-report-plugin")
+}
+
+dependencies {
+  api(project(":entity-service-api"))
+  api(project(":entity-service-change-event-api"))
+  implementation("com.typesafe:config:1.4.1")
```

Comment:
config is required to instantiate it, so should be part of public api rather than implementation.